apps/beeswax/src/beeswax/conf.py

```diff
@@ -123,6 +123,12 @@ def get_hive_thrift_http_port():
   dynamic_default=get_hive_thrift_http_port,
   type=int)
 
+HPLSQL = Config(
+  key="hplsql",
+  default=False,
+  type=coerce_bool,
```

Comment:
Is there a way to auto configure to true by reading a property in the hive-site.xml?